client/state/notices/middleware.js

```diff
@@ -128,21 +131,30 @@ export function onPostRestoreFailure( dispatch, action, getState ) {
 	dispatch( errorNotice( message ) );
 }
 
-export function onPostSaveSuccess( dispatch, action ) {
-	let text;
-	switch ( action.post.status ) {
+export function onPostSaveSuccess( dispatch, { post, savedPost }, getState ) {
+	switch ( post.status ) {
 		case 'trash':
-			text = translate( 'Post successfully moved to trash' );
+			const state = getState();
+			const userId = getCurrentUserId( state );
+			const isAuthor = savedPost.author && savedPost.author.ID === userId;
+			const canRestore = canCurrentUser(
+				state,
+				savedPost.site_ID,
+				isAuthor ? 'delete_posts' : 'delete_others_posts'
+			);
+
+			dispatch(
+				successNotice( translate( 'Post successfully moved to trash.' ), {
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'Post successfully moved to trash' )` *ES Score: 17*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/editor/delete-post-confirmation)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).